lib/ae_page_objects/element_proxy.rb

```diff
@@ -99,6 +99,22 @@ def method_missing(name, *args, &block)
       end
 
       implicit_element.__send__(name, *args, &block)
+    rescue Selenium::WebDriver::Error::StaleElementReferenceError
```

Comment:
Normally we are forced to reload the appropriate component in the test.  Am I correct in understanding you're removing the need for that?

Reply:
Yes, this has been a source of flakiness in tests since forever.  Hopefully we should see a reduction in flakiness with this change.